src/credentials/attestation_verifier/DeviceAttestationVerifier.h

```diff
@@ -318,10 +317,16 @@ class DeviceAttestationVerifier
             }
         }
 
+        VendorId BasicInformationVendorId() const { return mBasicInformationVendorId; }
+
+        uint16_t BasicInformationProductId() const { return mBasicInformationProductId; }
+
     private:
         Platform::ScopedMemoryBufferWithSize<uint8_t> mPaiDerBuffer;
         Platform::ScopedMemoryBufferWithSize<uint8_t> mDacDerBuffer;
         Platform::ScopedMemoryBufferWithSize<uint8_t> mCdBuffer;
+        VendorId mBasicInformationVendorId;
```

Comment:
This should be a uint16_t. Overall, we have to kill "VendorId" strongly typed enum --> it's dirty that we have to constantly deal with unknown values.

Reply:
+1 ... it is an odd one: https://github.com/project-chip/connectedhomeip/blob/master/src/lib/core/CHIPVendorIdentifiers.hpp#L42

it seems somewhat arbitrary on what is or is not there. I would prefer a typedef of uint16_t and then have some constants for common and notspecified.

Also a ??? towards  having `Common == Unspecified == 0` and another ?? on having `Unspecified` and `NotSpecified` with different (!) constant values.

Reply:
Also why is this using a `.hpp` extension instead of `.h` (took me a while to find because of that)

Reply:
I can make this a uint16_t, sure.  Fixing VendorId generally is https://github.com/project-chip/connectedhomeip/issues/21459 and I'd rather not try to do it here.